src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -99,6 +109,21 @@ It supports the following flags:
 
 - `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html).
 
+### GCP specific
+
+- `--gcp-project` - The Google Cloud Platform project where the cluster will be deployed
+- `--region` - The Google Cloud Platform region where the cluster will be deployed
+- `--gcp-failure-domains` - The Google Cloud Platform zones where the cluster's control-plane nodes will be deployed.
+- `--gcp-control-plane-sa-email` (optional) - The Google Cloud Platform Service Account which the control-plane nodes will use (default: "default")
```

Comment:
If the default is `default`, is really optional? Would `default` work as a valid GCP SA?

I was wondering if we should also tell which roles/permissions are needed.

Reply:
It does work AFAIK, I put "default" because it's what [the GCPMachineTemplate has as a default value](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/42f4747fc2b4181ba4898b1b3f3107718a8f5ac2/config/crd/bases/infrastructure.cluster.x-k8s.io_gcpmachinetemplates.yaml#L145-L147). This **can** work if the default compute service account has sufficient permissions. [It will have Editor permissions after you create a project.](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account)
So in the general case `default` should be alright here.